libs/cri-storing-service/src/main/java/uk/gov/di/ipv/core/library/cristoringservice/CriStoringService.java

```diff
@@ -149,6 +149,19 @@ public void storeVcs(
                 new AuditEventUser(
                         userId, ipvSessionItem.getIpvSessionId(), govukSigninJourneyId, ipAddress);
 
+        var sessionVcs =
+                sessionCredentialsService.getCredentials(
+                        ipvSessionItem.getIpvSessionId(), userId, true);
+
+        var scopeClaims = clientOAuthSessionItem.getScopeClaims();
+        for (var sessionVc : sessionVcs) {
+            if (!scopeClaims.contains(ScopeConstants.REVERIFICATION)) {
```

Comment:
maybe we could wrap this whole thing in `if (!scopeClaims.contains(ScopeConstants.REVERIFICATION))`?

Reply:
might be a q for Joe, but do you know if we also want to send all VCs when we call TICF?